google-interface/src/credentials/auth.js

```diff
@@ -84,22 +74,21 @@ function getNewToken(oAuth2Client, userTokenPath, resolve, reject) {
     output: process.stdout,
   })
 
-  rl.question('Enter the code from that page here: ', (code) => {
+  return rl.question('Enter the code from that page here: ', (code) => {
```

Comment:
`rl.question()` does not return a Promise, but we still need to return one (with the OAuth2Client object as the "context"). So we need to revert to using `return new Promise((resolve, reject) => ...)`

Reply:
Humm, so much time that I don't use callbacks, for me the function returned the result from the callback, from the [source](https://github.com/nodejs/node/blob/master/lib/readline.js#L293) of `question`, this is not the case. Fixing.